manticore/ethereum.py

```diff
@@ -697,14 +735,15 @@ def _compile(source_code, contract_name):
 
             if contract['bin'] == '':
                 raise Exception('Solidity failed to compile your contract.')
-                
-            bytecode = contract['bin'].decode('hex')
+
+            bytecode = ManticoreEVM._link(contract['bin'], libraries).decode('hex')
```

Comment:
`str.decode` doesn't exist in Py3 anymore. Use `import codecs` and `codecs.decode`

Reply:
I think `bytes.fromhex` is what we've been using in the py3 port branch